app/views/editor/_toolbar.html.erb

```diff
@@ -94,23 +94,23 @@
   <div class="btn-group mr-2 ">
     <a
       id="save-button-<%= toolbar_element_id %>"
-      data-toggle="tooltip" 
+      class="save-button btn btn-outline-secondary btn-sm"
       title="Save" 
+      data-form-id="<%= toolbar_element_id %>" 
       data-placement="bottom" 
-      onClick="$E.save()" 
-      class="save-button btn btn-outline-secondary btn-sm"
+      data-toggle="tooltip" 
     >
       <i class="fa fa-save"></i>
     </a>
     <a 
       id="recover-button-<%= toolbar_element_id %>"
-      data-toggle="tooltip" 
+      class="recover-button btn btn-outline-secondary btn-sm"
       title="Recover" 
+      data-form-id="<%= toolbar_element_id %>" 
       data-placement="bottom" 
-      onClick="$E.recover()" 
-      class="recover-button btn btn-outline-secondary btn-sm"
+      data-toggle="tooltip" 
     >
-      Recover
+      <i class="fas fa-undo"></i>
```

Comment:
New icon for the Recover Button!

Now it looks like this:
<img width="388" alt="Screen Shot 2021-02-04 at 8 13 18 PM" src="https://user-images.githubusercontent.com/4361605/106988858-a1be8280-6725-11eb-87a1-2e4c445a2d76.png">

Before (bleh):
<img width="423" alt="Screen Shot 2021-02-04 at 8 14 24 PM" src="https://user-images.githubusercontent.com/4361605/106988875-ac791780-6725-11eb-8d96-e33ff33fa8b7.png">